mmv1/products/redis/api.yaml

```diff
@@ -394,3 +394,57 @@ objects:
           - :READ_REPLICAS_DISABLED
           - :READ_REPLICAS_ENABLED
         input: true
+      - !ruby/object:Api::Type::NestedObject
+        name: persistenceConfig
+        description: Configuration of the persistence functionality.
+        min_version: beta
+        allow_empty_object: true
+        send_empty_value: true
+        properties:
+        - !ruby/object:Api::Type::Enum
+          name: persistenceMode
```

Comment:
This appears to have a hard default of `DISABLED`. That means we either need to set a default here, or use `default_from_api:true` as an override on the field.

Reply:
This comment still applies

Reply:
I'm not sure how to do it but I've switched from a `default_value: :DIRECT_PEERING` instead of the override in terraform.yaml: 
```yaml
      persistenceConfig.persistenceMode: !ruby/object:Overrides::Terraform::PropertyOverride
        default_from_api: true
```